open-metadata-implementation/admin-services/admin-services-server/src/main/java/org/odpi/openmetadata/adminservices/OMAGServerSecurityOfficerService.java

```diff
@@ -108,9 +93,12 @@ public VoidResponse setSecurityOfficerConfig(String userId, String serverName, S
         return response;
     }
 
-    private String getOutputTopicName(String securityServerType)
-    {
-        return outputTopic + securityServerType + defaultOutTopic;
+    private String getOutputTopicName(String securityServerType) {
+        if (securityServerType != null) {
+            return outputTopic + securityServerType + defaultOutTopic;
+        }
+
+        return outputTopic + "SecurityOfficerServer" + defaultInTopicName;
```

Comment:
Can I ask why you need defaultInTopicName here?

Reply:
There is no need. I corrected this issue in the last commit. Thank you!